services/src/main/java/io/grpc/protobuf/services/ProtoReflectionService.java

```diff
@@ -68,11 +68,8 @@
 @ExperimentalApi("https://github.com/grpc/grpc-java/issues/2222")
 public final class ProtoReflectionService extends ServerReflectionGrpc.ServerReflectionImplBase {
 
-  private final Object lock = new Object();
```

Comment:
Why get rid of this lock? We generally do this on purpose to avoid letting a user from interacting with our synchronization.

Reply:
Added back.